### PR TITLE
Make the README use a single type in examples.

### LINKED ISSUE #25978: [DOCS] Examples contained in README.textile are no longer valid
In the current [README.textile](https://github.com/elastic/elasticsearch/blob/master/README.textile) there are a few references to multiple types per index. In the example below there are 2 types ( `user` and `tweet` ) on the index `twitter` : 

```
curl -XPUT 'http://localhost:9200/twitter/user/kimchy?pretty' -H 'Content-Type: application/json' -d '{ "name" : "Shay Banon" }'

curl -XPUT 'http://localhost:9200/twitter/tweet/1?pretty' -H 'Content-Type: application/json' -d '
{
    "user": "kimchy",
    "post_date": "2009-11-15T13:12:00",
    "message": "Trying out Elasticsearch, so far so good?"
}'
```
As per #24428 only one type can be created per index, and the above requests result in an error :
```
"type" : "illegal_argument_exception",
"reason" : "Rejecting mapping update to [twitter] as the final mapping would have more than 1 type: [tweet, user]"
``` 
Relates to #25401

### PR DESCRIPTION
**Pull request by jpountz**
It currently indexes both users and tweets as their own types.

Closes #25978

### CODE REVIEW

File: README.textile [hunk 1]
```diff
@@ -50,17 +49,19 @@ h3. Indexing
 Let's try and index some twitter like information. First, let's create a twitter user, and add some tweets (the @twitter@ index will be created automatically):
 
 <pre>
-curl -XPUT 'http://localhost:9200/twitter/user/kimchy?pretty' -H 'Content-Type: application/json' -d '{ "name" : "Shay Banon" }'
+curl -XPUT 'http://localhost:9200/twitter/doc/user_kimchy?pretty' -H 'Content-Type: application/json' -d '{ "type": "user", "name" : "Shay Banon" }'
```
Comment (dadoonet) [inline README.textile]: In real life, would we create instead 2 indices?

```sh
curl -XPUT 'http://localhost:9200/twitter_user/doc/kimchy
curl -XPUT 'http://localhost:9200/twitter_tweet/doc/1
```

**Review by dadoonet — COMMENTED** [review]
It looks good to me but I think we should may be not promote using different type of data within the same index.

WDYT?

**Comment by jpountz** [comment]
I did not do that in order to keep changes minimal but I think you are right that we should use different indices for tweets and users, I'll fix.

**Comment by jpountz** [comment]
@dadoonet I ended up only having one index for tweets to keep things simple, can you have another look?

**Review by dadoonet — COMMENTED** [review]
Left some other comments.

File: README.textile [hunk 2]
```diff
@@ -125,29 +129,27 @@ h3. Multi Tenant - Indices and Types
...
+curl -XPUT 'http://localhost:9200/kimchy/doc/2?pretty' -H 'Content-Type: application/json' -d '
 {
     "user": "kimchy",
     "post_date": "2009-11-15T14:12:12",
     "message": "Another tweet, will it be indexed?"
 }'
 </pre>
 
-The above will index information into the @kimchy@ index, with two types, @info@ and @tweet@. Each user will get their own special index.
+The above will index information into the @kimchy@ index. Each user will get their own special index.
```
Comment (jpountz) [inline README.textile]: It looks ok to me?

Comment (dadoonet) [inline README.textile]: My bad.

**Review by dadoonet — APPROVED** [review]
LGTM!

**Pull request merged**